internal/server/rpc_guest.go

```diff
@@ -19,6 +19,8 @@ func (ts *TerminalServer) TerminalChannel(channel api.GuestService_TerminalChann
 		return status.Errorf(codes.FailedPrecondition, "expected a Hello message")
 	}
 
+	ts.logger.Infof("Got hello from %s", helloFromGuest.Locator)
```

Comment:
You sure we need this here?

Reply:
I think it's nice for debugging. 🤷‍♂️ Let's keep it. It's not too spammy